app/views/about/contact.haml

```diff
@@ -0,0 +1,12 @@
+#contact-content.odd{style: 'padding: 15px;'}
+  %h1 Contact the Constellation
+
+  %p
+    Currently, to contact the Constellation for help or to report a violation of our
+    = succeed ',' do
+      = link_to 'Terms of Service', tos_path
+    you may visit us in the #constellation channel of the
+    = link_to "Glowfic Discord", 'https://discord.gg/Ss6fRFj', target: '_blank'
```

Comment:
Is this discord link the same as we have on the homepage? If so, we might want to extract it somewhere so it's easier to amend in the future if/when we need to.